tests/integration_tests/style/test_json.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for JSON linter checks."""
+
+import framework.utils as utils
```

Comment:
This should be placed after "import json" or otherwise our python linter will complain.